website/blog/2023-09-05-wrapper-as-a-standalone-component.md

```diff
@@ -0,0 +1,157 @@
+---
+title: How to use the Wrapper as a standalone component
+description: Explains the process of using the Wrapper component in WordPress templates.
+slug: wrapper-as-a-standalone-component
+authors: kancijan
+
+date: 2023-09-05
+tags: [eightshift, boilerplate, wrapper, components]
+hide_table_of_contents: false
+---
+
+As one of the most powerful features in the Eightshift DevKit, the Wrapper is a component used for every block in the Gutenberg editor, but what about WordPress templates?
+<!--truncate-->
+
+:::note
+[The Wrapper](https://eightshift.com/docs/basics/blocks-wrapper/) is designed to be the ultimate top-level component that controls how your block behaves in the website layout. The Wrapper is a sort of a 'section' in traditional builders. By default, you can control a whole lot of stuff, but there is an option to add your custom attributes and fine-tune the Wrapper to your project's needs.
+:::
+
+## What are WordPress templates?
+
+Before the days of [Full Site Editing](https://developer.wordpress.org/block-editor/getting-started/full-site-editing/) in WordPress, we used [template files](https://developer.wordpress.org/themes/basics/template-hierarchy/) to add custom layout to a blog page or post archives.
```

Comment:
Maybe we could mention that we're not FSE-ready yet?

Reply:
https://eightshift.com/docs/basics/blocks-faq/#will-this-setup-work-with-full-site-editing

Reply:
It won't work as-is, the theme would need a bit of modifications to work with FSE 😄

Reply:
something like... ?
:::caution Important
Eightshift Boilerplate (v6.x.x) is not yet FSE ready.
:::